.github/workflows/update-labels.yml

```
@@ -1,78 +1,58 @@
 on:
-  workflow_run:
-    workflows:
-      - Remove and Add Multiple Labels on Approval
+  issue_comment:
     types:
-      - completed
+      - created
 
 jobs:
   update-labels:
+    if: contains(github.event.comment.body, 'PR approved: Please update labels.')
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 10: mapping values are not allowed in this context

(syntax-check)